src/Compilers/Core/CodeAnalysisTest/AnalyzerFileReferenceTests.cs

```diff
@@ -187,17 +187,23 @@ public void TestLoadErrors3()
             Assert.Equal(0, errors.Count);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/6665")]
+        [Fact]
         public void TestAnalyzerLoading()
         {
             var dir = Temp.CreateDirectory();
             var test = dir.CopyFile(typeof(FromFileLoader).Assembly.Location);
             var analyzerFile = TestHelpers.CreateCSharpAnalyzerAssemblyWithTestAnalyzer(dir, "MyAnalyzer");
```

Comment:
That's actually not required as [Temp](http://source.roslyn.io/Roslyn.Test.Utilities.Desktop/R/b2f9b4034b00b7f5.html) will get disposed after each test class finishes executing, which deletes the temp directory created in the tests.